src/pages/features/index.md

```diff
@@ -235,6 +247,9 @@ If the use case would be to be able to create an `.xmp` file from a set of slide
 1. Start with the empty `.xmp` file from [here](https://github.com/AdobeDocs/cis-photoshop-api-docs/blob/main/sample-code/lr-sample-app/crs.xml) and add the corresponding slider values
 2. Or please look ahead in this documentation page at the [/edit API](/features/#edit)
 
+Example of Presets with a sample image. Presets for Aged Photo is applied on the original image.
```

Comment:
In this example, we are applying the Preset called "Aged Photo" to automatically make the adjustments.